platform.yaml

```diff
@@ -47,7 +47,6 @@ substitutions:
   topic_outdoor:            'pub/xxx/xxx/xxx/temperature' #if you will use IOT guru, please uncomment and update this to get graphs working
 
 # MQTT topics. Notice that these are inside both single and double quotes:
-  topic_delta:                      '"pub/xxx/xxx/xxx/delta"'
```

Comment:
Do not remove this. Will be still used. But " characters needs to be removed and leave just ' s